src/main/java/pages/CategoryPage.java

```diff
@@ -0,0 +1,68 @@
+package pages;
+
+import browser.BrowserImpl;
+import browser.BrowserSize;
+import logger.MyLogger;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.support.FindBy;
+import utils.WaitHelper;
+import webelements.elements.Link;
+import webelements.elements.WebElements;
+
+public class CategoryPage extends BasePage {
+    @FindBy(xpath = "//a[@href='/Mens-Clothing/bc_2___.htm']")
+    private Link categoryMen;
+    @FindBy(xpath = "//div[@class='mn_categoryPage']")
+    private WebElements page;
+    @FindBy(xpath = "//span[@class = 'mn_mainNavTrigger']")
+    private WebElements navArrow;
+    @FindBy(xpath = "//button[@class='mn_navTrigger americanairlines-svgIcon americanairlines-icon_hamburger']")
+    private WebElements hamburgerIcon;
+    @FindBy(xpath = "//span[@role='button'][@class='mn_mainNavTopItem' and @aria-expanded='false']")
+    private WebElements mobileCategoriesNav;
+    @FindBy(tagName = "select")
+    private WebElements selectDropDown;
+
+    private  BrowserImpl browser = new BrowserImpl();
+
+    public void clickOnCategoryFromDesktopNav(){
```

Comment:
too many for PO actions, don't put choosing browser into PO, it's a BO responsibility